### ISSUE TITLE
request for vertical swipe between images

### ISSUE THREAD

**Issue by sirgreeno** [brutaldesign/swipebox#209]
Would it be possible to adjust the swipebox navigation so that instead of horizontally swiping to switch between images, you could swipe from top-bottom/bottom-top? If so, how would this be done?

**Comment by Download** [comment]
Yes this is possible, but it would require a change to the code.
Have a look at function [`gesture`](https://github.com/brutaldesign/swipebox/blob/master/src/js/jquery.swipebox.js#L278), it contains the code to determine whether a vertical or a horizontal swipe is happening. Specifically, on line [312](https://github.com/brutaldesign/swipebox/blob/master/src/js/jquery.swipebox.js#L312) it binds a listener to `touchmove`:
```javascript
$( '.touching' ).bind( 'touchmove',function( event ) {
```
Inside that listener, it sets properties `hSwipe` and `vSwipe` based on whether it detected a **h**orizontal or **v**ertical swipe, respectively. Then, on line [342](https://github.com/brutaldesign/swipebox/blob/master/src/js/jquery.swipebox.js#L342) it checks whether swipe was horizontal and does it's thing:

```javascript
if ( hSwipe ) {
	// swipe left
	if ( 0 < hDistance ) {
		// ...
	// swipe rught
	} else if ( 0 > hDistance ) {
		// ...
	}
}

Do note that currently, vertical swipe is used to close the Swipebox. This can be seen in line [381](https://github.com/brutaldesign/swipebox/blob/master/src/js/jquery.swipebox.js#L381) and beyond, where it attaches a listener to `touchend`:

```javascript
} ).bind( 'touchend',function( event ) {
```

Inside this listener it checks for and handles both `hSwipe` (for **h**orizontal swipes) and `vSwipe` (for **v**ertical swipes).

So, to make the whole thing work the other way around and move to the prev/next slide on vertical swipes and close the Swipebox on horizontal swipes, you would just change around `vSwipe` and `hSwipe` in the logic in the listener for `touchend`.

If you want to make this configurable, you can add a setting to the `defaults` and listen to that (line [9](https://github.com/brutaldesign/swipebox/blob/master/src/js/jquery.swipebox.js#L9)):
```javascript
defaults = {
	// ....
	toggleClassOnLoad: '',
	swipeMode: 'horizontal',
	swipeToClose: 'true'
},
```

Then check the setting before deciding whether to react to `vSwipe` or `hSwipe`.

Good luck!
